### PR TITLE
Padding fixes, markets data null fix, update .env.example

### PR DESCRIPTION
**Pull request by aforaleka**
- update .env.example for token addresses / migration url
image

- bump default icon size in icon button
image

- fix padding in rewards/ fees page
image
image

- fix fees table overflow on mobile
image

- fix market details overflow on mobile
image

- panel improvements
image

- remove notifications menu close button on mobile
image

- fix markets data crashing
- update ToggleGroup logic, fix markets dropdown crashing when filter is unselected
- fix nested button console warning for dropdown header menu on mobile

### CODE REVIEW

File: src/components/Panel.tsx [hunk 1]
```diff
@@ -53,19 +53,30 @@ export const Panel = ({
...
   ${layoutMixins.row}
 
   background-color: var(--color-layer-3);
   border-radius: 0.875rem;
+
+  ${({ onClick }) =>
+    onClick &&
+    css`
+      cursor: pointer;
+    `}
```
Comment (jaredvu) [inline src/components/Panel.tsx:79]: Should we also add `hover/focus` styling here? If there is an easy way to display the arrow's `:hover` effect when the parent `<Panel>` is hovered that would be ideal.

Screen Shot 2023-10-31 at 10 35 32 AM

File: src/views/MarketFilter.tsx [hunk 2]
```diff
@@ -29,12 +29,14 @@ export const MarketFilter = ({
...
         <ToggleGroup
-          items={Object.entries(filters).map(([key, value]) => ({
+          items={Object.values(filters).map((value) => ({
             label: stringGetter({ key: MARKET_FILTER_LABELS[value] }),
             value,
           }))}
           value={selectedFilter}
-          onValueChange={onChangeFilter}
+          onValueChange={(newFilter: string) => {
+            onChangeFilter((newFilter || selectedFilter) as MarketFilters);
```
Comment (jaredvu) [inline src/views/MarketFilter.tsx:38]: is `newFilter` ever expected to be empty or nullable?

Comment (aforaleka) [reply src/views/MarketFilter.tsx:38]: that's how ToggleGroup is set up currently, when you click on it again, it "removes" the filter

File: src/pages/portfolio/Fees.tsx [hunk 3]
```diff
@@ -56,28 +55,28 @@ export const Fees = () => {
...
         )}
         {!!makerShare && (
-          <Styled.TextRow>
-            {isAdditional && <span>{stringGetter({ key: STRING_KEYS.AND })}</span>}
-            <span>{stringGetter({ key: STRING_KEYS.MAKER_MARKET_SHARE })}</span>
-            <Styled.Highlighted>{'>'}</Styled.Highlighted>
+          <Styled.AdditionalConditionsText>
+            {isAdditional && stringGetter({ key: STRING_KEYS.AND })}{' '}
+            {stringGetter({ key: STRING_KEYS.MAKER_MARKET_SHARE })}{' '}
+            <Styled.Highlighted>{' >'}</Styled.Highlighted>{' '}
```
Comment (jaredvu) [inline src/pages/portfolio/Fees.tsx:73]: is the additional space intentional? I don't think it changes the rendering

**Review by jaredvu — APPROVED** [review]

**Pull request merged**